config/settings.yml

```diff
@@ -186,6 +185,17 @@ preneeds:
   wsdl: 'config/preneeds/wsdl/preneeds.wsdl'
   burial_form_schema: 'config/preneeds/burial_form_schema.json'
 
+# Settings for VBA Document upload service module
+vba_documents:
+  location:
+    prefix:  http://some.fakesite.com/path
```

Comment:
TODO settings documentation, maybe some additional comments here for behavior?

Reply:
I think @kreek has something in mind for per-module settings that may change how this is done.